client/verta/verta/_registry/modelversion.py

```diff
@@ -794,6 +794,9 @@ def get_attributes(self):
         self._refresh_cache()
         return _utils.unravel_key_values(self._msg.attributes)
 
+    def _get_attribute_keys(self):
+        return map(lambda attribute: attribute.key, self.get_attributes())
```

Comment:
To match `get_artifact_keys()`
```suggestion
        return list(map(lambda attribute: attribute.key, self.get_attributes()))
```